### PR TITLE
v5.0.0-beta.0 – Initial v5 work to move all styles into optional mixins

### PR DESCRIPTION
**Pull request by ashleynolan**
**_Documentation/wiki updates will be done prior to full v5 release_**

### Added
- Base template includes for legacy consuming applications (PoC – needs testing).

### Changed
- Made all styles and components optional (wrapped in a mixin).

### Removed
- `/settings/glyphs` moved into badges, as it is the only place it is used.
- `/images` as not used.

## UI Review Checks

- [x] This PR has been checked with regard to our brand guidelines

### CODE REVIEW

File: src/scss/base/_layout.scss [hunk 1]
```diff
@@ -1,137 +1,142 @@
...
+    // utility that lets you align items side by side and vertically centered
+    %l-inlined,
+    .l-inlined {
+        display: flex;
+        align-items: center;
 
-    .no-flexbox & {
-       display: inline-block;
+        .no-flexbox & {
+        display: inline-block;
```
Comment (kevinrodrigues) [inline src/scss/base/_layout.scss:126]: Minor tabbing issue here.

File: src/scss/components/_breadcrumbs.scss [hunk 2]
```diff
@@ -1,132 +1,137 @@
...
+    .c-breadcrumb-item-link {
+        text-decoration: none;
+        color: $grey--darkest;
+        font-weight: $breadcrumb-font-weight;
 
-    .c-breadcrumb-item {
-        color: $white;
-        svg {
-            fill: $white;
+        &:hover {
```
Comment (kevinrodrigues) [inline src/scss/components/_breadcrumbs.scss:42]: `&:focus`?

File: src/scss/components/_media-element.scss [hunk 3]
```diff
@@ -1,173 +1,177 @@
...
+            right: $mediaElement-spacing;
+            text-decoration: none;
+            top: $mediaElement-spacing;
+
+            @include media('<mid') {
+                right: spacing(x2);
+                top: spacing(x2);
+            }
+
+            &:hover {
```
Comment (kevinrodrigues) [inline src/scss/components/_media-element.scss:173]: `:focus`?

**Review by kevinrodrigues — COMMENTED** [review]
LGTM 👍 minor comments.

File: src/scss/base/_layout.scss (hunk 1 above)
Comment (ashleynolan) [reply src/scss/base/_layout.scss]: Fixed

File: src/scss/components/_breadcrumbs.scss (hunk 2 above)
Comment (ashleynolan) [reply src/scss/components/_breadcrumbs.scss:42]: If it's ok, I'll leave this as is (as I was only changing the indentation in these files). I'll test it in a subsequent PR though to see if it needs the focus state.

File: src/scss/components/_media-element.scss (hunk 3 above)
Comment (ashleynolan) [reply src/scss/components/_media-element.scss:173]: Same as above

**Review by kevinrodrigues — APPROVED** [review]

File: src/scss/components/optional/_apps-banner.scss [hunk 4]
```diff
@@ -1,15 +1,16 @@
...
- * =================================
- * Used to display app promo banner on the page.
- *
- * *The `c-appBaner` component is an optional mixin within Fozzie — if you'd like to use it in your project you can include it by adding `@include appsBanner();` into your SCSS dependencies file.*
- *
- * Documentation:
- * https://fozzie.just-eat.com/styleguide/ui-components/apps-banner
- */
-
 @mixin appsBanner() {
```
Comment (DamianMullins) [inline src/scss/components/optional/_apps-banner.scss:1]: Now that all components are going to be optional do we still need the`/optional` directory?

Comment (ashleynolan) [reply src/scss/components/optional/_apps-banner.scss:1]: I'll be removing that in my next PR – didn't want to re-arrange it in this one as it kind of white-washes over the changes in files then in Github.

**Review by DamianMullins — APPROVED** [review]

File: src/scss/components/optional/_apps-banner.scss (hunk 4 above)
Comment (DamianMullins) [reply src/scss/components/optional/_apps-banner.scss:1]: Makes sense.

**Pull request merged**